notebooks/002-datamodules/002-btech.ipynb

```diff
@@ -0,0 +1,576 @@
+{
```

Comment:
<p>I think it would be good to mention that the PL datamodule is also fully compatible with the PL trainer class, so that if our entrypoint scripts or the anomalib CL are used for training, users don't have to bother with any data loading logic. Though maybe the readme in <code>notebooks/002-datamodules</code> would be a more appropriate place to mention this. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Not sure if I get this. Can you elaborate</p>